osgar/drivers/usbcam.py

```diff
@@ -35,6 +45,8 @@ def join(self, timeout=None):
         self.input_thread.join(timeout=timeout)
 
     def run_input(self):
+        if not hasattr(self, 'cap'):
```

Comment:
I would maybe initialize self.cap to None, but OK

Reply:
done